runners/apex/src/main/java/org/apache/beam/runners/apex/ApexPipelineOptions.java

```diff
@@ -62,4 +62,9 @@
   @Default.String("classpath:/beam-runners-apex.properties")
   String getConfigFile();
 
+  @Description("configure whether to perform ParDo fusion")
```

Comment:
I don't think this is needed. It should always be done. The user can tweak stream locality through the apex properties if needed.